### DISCUSSION TITLE
Sharepoint List of Files in a specific Folder

### DISCUSSION THREAD

**Discussion by runnerbr** [O365/python-o365#921] [Q&A]
Hello all,
I'm trying to list the files from a specific folder in Sharepoint.
I know it's possible to do that if I know the filename through get_item_by_path, but I need to list the files from the folder.
Can someone help me?

Best regards.
Thank you.

**Comment by alejcas** [comment]
Check drive.py source code . The answer is there

**Comment by guidorietbroek** [comment]
You can check out the resource from MS Graph, it will help you to get to your answer.

[Working with Sharepoint sites in Microsoft Graph](https://learn.microsoft.com/en-us/graph/api/resources/sharepoint?view=graph-rest-1.0)
